tuning/libtuner.py

```diff
@@ -184,113 +194,45 @@ class ParsedDisptachBenchmarkResult:
-        if self.get_model_candidate_path():
-            try:
-                path_str = self.get_model_candidate_path()
-                return int(path_str.split("_")[-1].split(".")[0]) if path_str else None
-            except ValueError:
-                return None
-        return None
-
-    def get_device_id(self) -> Optional[int]:
-        if len(self.get_tokens()) < 5:
+        pattern = r"process_time/real_time_mean\s+([\d.]+)\s\w{2}"
+        match = re.search(pattern, self.result_str)
+        if not match:
             return None
         try:
-            return int(self.get_tokens()[4])
+            return float(match.group(1))
         except ValueError:
             return None
 
-    def get_benchmark_time(self) -> Optional[int | float]:
-        if len(self.get_tokens()) < 7:
-            return None
-        try:
-            return float(self.get_tokens()[6])
-        except ValueError:
-            return None
 
-    def get_calibrated_result_str(self, change: float) -> Optional[str]:
-        if self.result_str is None:
-            return self.result_str
+def generate_display_DBR(
+    candidate_id: int = 0, mean_time: float = random.uniform(100.0, 500.0)
+) -> str:
+    """Generate dispatch_benchmark_result string for displaying"""
+    return f"{candidate_id}\tMean Time: {mean_time:.1f}\n"
 
-        benchmark_time = self.get_benchmark_time()
-        if benchmark_time is None:
-            return self.result_str
 
-        # Format the change to be added to the string
-        percentage_change = change * 100
+def generate_display_MBR(
+    candidate_vmfb_path_str: str = "baseline.vmfb",
+    device_id: str = "0",
+    t1: float = random.uniform(100.0, 500.0),
+    calibrated_diff: Optional[float] = None,
+) -> str:
+    """Generate model_benchmark_result string for displaying"""
+    if calibrated_diff:
+        percentage_change = calibrated_diff * 100
         change_str = f"({percentage_change:+.3f}%)"
-
-        # Use regex to find and replace the old benchmark time with the new one
-        new_result_str = re.sub(
-            r"(\d+(\.\d+)?)\s*ms",
-            lambda m: f"{self.get_benchmark_time()} ms {change_str}",
-            self.result_str,
-            count=1,
-        )
-
-        return new_result_str
-
-    def generate_sample_result(
-        self,
-        candidate_vmfb_path_str: str = "unet_baseline.vmfb",
-        device_id: int = 0,
-        t1: float = random.uniform(100.0, 500.0),  # time in ms
-    ) -> str:
-        return f"Benchmarking: {candidate_vmfb_path_str} on device {device_id}\nBM_run_forward/process_time/real_time_median\t    {t1:.3g} ms\t    {(t1+1):.3g} ms\t      5 items_per_second={t1/200:5f}/s\n\n"
+        res_str = f"Benchmarking: {candidate_vmfb_path_str} on device {device_id}: {t1:.3g} {change_str}\n\n"
+    else:
+        res_str = f"Benchmarking: {candidate_vmfb_path_str} on device {device_id}: {t1:.3g}\n\n"
+    return res_str
```

Comment:
```suggestion
        return f"Benchmarking: {candidate_vmfb_path_str} on device {device_id}: {t1:.3g} {change_str}"
   
    return f"Benchmarking: {candidate_vmfb_path_str} on device {device_id}: {t1:.3g}"
```

and add the newlines in the code that uses this function